passes/cmds/select.cc

```diff
@@ -1113,6 +1113,9 @@ struct SelectPass : public Pass {
 		log("    <obj_pattern>\n");
 		log("        select the specified object(s) from the current module\n");
 		log("\n");
+		log("Prefix the following patterns with '=' if the pattern should match black-/\n");
+		log("white-box modules and their contents.\n");
```

Comment:
That inversion of the previous statement is at least misleading. The this is written it suggests that `=` selects only black-/white-box module. But it selects them in addition to other modules.

Reply:
Good point. See new suggestion.